recipes-support/blink-led/blink-led.bb

```diff
@@ -0,0 +1,32 @@
+SUMMARY = "LED blink test"
+LICENSE = "MIT"
+
+LIC_FILES_CHKSUM = "file://${COREBASE}/meta/COPYING.MIT;md5=3da9cfbcb788c80a0384361b4de20420"
+
+SRC_URI = " \
+    file://blink.service \
+    file://blink-led \
+"
+
+SYSTEMD_SERVICE_${PN} = "blink.service"
+
+BLINK_COLOR = "blue"
+
+
+do_install() {
+
+    install -d ${D}/${bindir}
+    if [ ${BLINK_COLOR} = "blue" ];
+    then
+        sed -i -e '0,/red/ s/red/blue/' ${WORKDIR}/blink-led
+    fi
```

Comment:
Instead of using an if-statement here, consider removing it and simply parse the variable `BLINK_COLOR` directly using `sed`.